README.md

```diff
@@ -19,123 +21,19 @@ production ready.
 
 Service is initially a product of the Guardian Discussion team.
 
-## Definition
-
-To qualify as a Service requires:
-
-* following certain AWS conventions
-* one-click deploys and stack creation
-* some specific documentation
-* sufficient monitoring and alerting
-
-Harder to quantify, but equally important, if not more so, is:
-
-* a decoupled architecture, including no hard-dependencies (graceful
-  failure)
-
-## Spec
+## Terminology
 
     MUST - required
     SHOULD - expected but not required
 
-### Naming conventions
-
-Components of names are:
-
-    org   -> the organisation producing the service, e.g. com.gu
-    stack -> conceptual group the app belongs too
-    app   -> the name of the service itself
-    STAGE -> the environment the app is running in (e.g. PROD), must
-             be capitalised
-
-The general pattern is:
-
-    [stack]-[app]-[STAGE]
-
-(Clauses separated by hyphens, and the stage capitalised.)
-
-In certain cases, further information may be appended onto the end of
-this pattern as appropriate. For example, a Cloudwatch alarm might be
-called `discussion-search-PROD-disk-util`.
-
-#### S3
-
-S3 names are global. As a result, they should be prefaced with
```

Comment:
Shouldn't S3 names also contain the STAGE? You wouldn't want to share the same bucket for PROD and CODE would you?